README.md

```diff
@@ -1,64 +1,66 @@
 
 <a href="https://badge.fury.io/js/ignite-cli" target="_blank"><img src="https://badge.fury.io/js/ignite-cli.svg" alt="npm version" height="20"></a>
 [![CircleCI](https://dl.circleci.com/status-badge/img/gh/infinitered/ignite/tree/master.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/infinitered/ignite/tree/master)
 
 ## Battle-tested React Native boilerplate
 
-The culmination of five years of constant React Native development, Ignite is the most popular React Native app boilerplate for both Expo and bare React Native.
+The culmination of over six years of constant React Native development, Ignite is the most popular React Native app boilerplate for both Expo and bare React Native.
 
 This is the React Native boilerplate that the [Infinite Red](https://infinite.red) team uses on a day-to-day basis to build client apps. Developers who use Ignite report that it saves them two to four weeks of time on average off the beginning of their React Native project!
 
 ## Tech Stack
 
-Ignite apps include the following rock-solid technical decisions out of the box:
-
-- React Native
-- React Navigation 6
-- MobX-State-Tree [(Why not Redux?)](https://github.com/infinitered/ignite/blob/master/docs/MobX-State-Tree.md)
-- MobX-React-Lite
-- TypeScript
-- AsyncStorage (integrated with MST for restoring state)
-- apisauce (to talk to REST servers)
-- Flipper-ready
-- Reactotron-ready (and pre-integrated with MST)
-- Supports Expo (and Expo web) out of the box
-- Hermes enabled
-- React native Reanimated
-- And more!
+Nothing makes it into Ignite unless it's been proven on projects that Infinite Red works on. Ignite apps include the following rock-solid technical decisions out of the box:
+
+| Library           | Category             | Version | Description                                    |
+| ----------------- | -------------------- | ------- | ---------------------------------------------- |
+| React Native      | Mobile Framework     | v0.69   | The best cross-platform mobile framework       |
+| React             | UI Framework         | v18     | The most popular UI framework in the world     |
+| TypeScript        | Language             | v4      | Static typechecking                            |
+| React Navigation  | Navigation           | v6      | Performant and consistent navigation framework |
+| MobX-State-Tree   | State Management     | v5      | Observable state tree                          |
+| MobX-React-Lite   | React Integration    | v3      | Re-render React performantly                   |
+| Expo              | SDK                  | v46     | Allows (optional) Expo modules                 |
+| Expo Font         | Custom Fonts         | v10     | Import custom fonts                            |
+| Expo Localization | Internationalization | v13     | i18n support (including RTL!)                  |
+| Expo Status Bar   | Status Bar Library   | v1      | Status bar support                             |
```

Comment:
I left the Expo Splash Screen one off because you only use it if you're using Expo; otherwise, you use RN Boot Splash. (Would be nice if we could use it for both...)